Gemfile.lock

```diff
@@ -198,4 +198,4 @@ RUBY VERSION
    ruby 2.3.1p112
 
 BUNDLED WITH
-   1.16.3
+   1.16.2
```

Comment:
I'm a bit worried about the updates. Is the site working properly on your localhost?

Reply:
Hey so sorry, I only saw this comment, the localhost is working fine on my computer.